### PR TITLE
Xcode12

### PR DESCRIPTION
**Pull request by nchatharoo**

### CODE REVIEW

**Comment by caiozullo** [comment]
Hi @nchatharoo, please rename the PR following the instructions in the readme file. Thank you.

File: FeedAPIChallenge/Feed API/FeedImagesMapper.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Chatharoo on 24/01/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImagesMapper {
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed API/FeedImagesMapper.swift:11]: The default access level is internal so the `internal` keyword can be omitted.

File: FeedAPIChallenge/Feed API/FeedImagesMapper.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		let image_desc: String?
+		let image_loc: String?
+		let image_url: URL
+		
+		var image: FeedImage {
+			return FeedImage(id: image_id, description: image_desc, location: image_loc, url: image_url)
+		}
+	}
+		
+	internal static func map(_ data: Data, from response: HTTPURLResponse) -> RemoteFeedLoader.Result {
```
Comment (mapostolakis) [inline FeedAPIChallenge/Feed API/FeedImagesMapper.swift:31]: The default access level is internal so the `internal` keyword can be omitted.

**Review by mapostolakis — COMMENTED** [review]
Hi @nchatharoo,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs.

I left a couple of comments about omitting the `internal` keyword. Feel free to push a new commit.

Great job! ✅

**Pull request closed without merging**